### PR TITLE
EOS-19737 : Support for Reset & Cleanup phase in utils_Setup

### PR DESCRIPTION
**Pull request by vastradparayya**
This provides reset/cleanup interface

Jira ticket reference - [EOS-19737](https://jts.seagate.com/browse/EOS-19737)

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: py-utils/src/setup/utils.py [hunk 1]
```diff
@@ -160,5 +159,33 @@ def test():
 
     @staticmethod
     def reset():
-        """ Performs Configuraiton reset """
+        """ Remove/Delete all the data that was created after post install """
+
+        import os
+        conf_file = "/etc/cortx/message_bus.conf"
+        if not os.path.exists(conf_file):
```
Comment (selvakumaar5496) [inline py-utils/src/setup/utils.py:166]: @sachinpunadikar , Does reset requires this check? Anyways messageBus throws an error if conf file does not exist.

Comment (sachinpunadikar) [reply py-utils/src/setup/utils.py:166]: @selvakumaar5496 , the MessageBus error will not provide the message (reset/cleanup already done), which we want to convey  .. rt ?

File: py-utils/src/setup/utils.py [hunk 2]
```diff
@@ -160,5 +159,33 @@ def test():
...
+        config_file = "/etc/cortx/message_bus.conf"
+        if not os.path.exists(config_file):
+            raise SetupError(errno.ENOENT, "Cleanup has already done or config file not found! ")
+
+        # delete data/config stored
+        cmd = f"rm -rf {config_file}"
+        cmd_proc = SimpleProcess(cmd)
+        _, res_err, res_rc = cmd_proc.run()
+        if res_rc != 0:
+            raise SetupError(errno.EIO, "Error while deleting config file %s", res_err)
```
Comment (ujjwalpl) [inline py-utils/src/setup/utils.py:190]: Please add the file as parameter that you are trying to delete.

File: py-utils/src/setup/utils.py [hunk 3]
```diff
@@ -160,5 +159,33 @@ def test():
...
+    def cleanup():
+        """ Cleanup message bus config and logs. """
+        import os
+        config_file = "/etc/cortx/message_bus.conf"
+        if not os.path.exists(config_file):
+            raise SetupError(errno.ENOENT, "Cleanup has already done or config file not found! ")
+
+        # delete data/config stored
+        cmd = f"rm -rf {config_file}"
+        cmd_proc = SimpleProcess(cmd)
```
Comment (ujjwalpl) [inline py-utils/src/setup/utils.py:187]: Why dont we simple os.remove() ?

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

**Pull request closed without merging**